setup.py

```diff
@@ -65,7 +65,7 @@ def get_version(version_tuple):
       long_description=LONG_DESCRIPTION,
       platforms=['any'],
       classifiers=CLASSIFIERS,
-      install_requires=['pymongo>=3.0,<3.14'],
+      install_requires=['pymongo>=3.7'],
```

Comment:
Is this change necessary? If so, why?

We're still going to run this code on PyMongo v3.13.0, which matched the previous constraints, right?

Reply:
`count_documents` and `estimated_document_count` were made available in pymongo 3.7. 

upper bound change reflects the support we now have for the next major

Reply:
I see, so the code as-is supports PyMongo v4 out of the box as well, yea? Then `pymongo>=3.7,<5.0` makes sense, since we don't know if v5+ will be supported – and we know it'll likely be released in the future, even if it's not released now.

Reply:
This was merged and it's not a big deal, but to be precise, I still think we should change `install_requires` to `pymongo>=3.7,<5.0`.